blog/steering/governance-changes.md

```diff
@@ -0,0 +1,42 @@
+| ------------- | ------------- | ------------- |
+| <!-- Your Github handle here --> |   |  |
+
+
+One of the long-standing [asks](https://github.com/knative/community/issues/33) within the Knative community has been for the Steering Committee to clarify its charter and how the committee is composed. I'm happy to be able to say that we now have a new [Steering Committee charter](https://github.com/knative/community/pull/277) which, in my opinion, delivers on this ask. Community participation was central to working through the process to get to this new charter, and I first and foremost want to say a sincere THANK YOU to everyone in our community that participated in our public discussions on this topic. I don't think it is an exaggeration to say that the entire community was involved in this effort, or that the community participation helped to influence the direction the discussions went.
+
+Let me summarize the meaning of the changes, and then go into some detail:
+
+- There will be elections for Steering Committee members
+- Steering committee members serve as individuals rather than representing their employer
+- No vendor will be allowed to have a majority of seats in the SC
+- We added the new Trademark Committee to handle matters related to the trademark
+- There is a transition plan from the current scheme to the new governance scheme
+
+Those of you who love committees will be pleased that we have increased the committee count within the project by one. Those of you who don't love committees for their own sake may be asking what the purpose of such a committee is; luckily, there is a clear and sensible answer. During the discussions around new governance, it became apparent that there was a clear separation of concerns between the specific vendor needs around trademark, "core" scope (for lack of a better term) and comformance, and the broader community needs around everything else Steering does. While we could have created a hybrid structure for Steering, we decided it was simpler to just have a vendor committee that focused only on scope and comformance, with the Steering committee focusing on the other aspects of community growth and project governance.
+
+The new "KTC" will initially include representatives from Google, Red Hat/IBM, and VMware. They will need to figure out how new components and features "plug in" to the Knative ecosystem in a non-technical sense. They will also help Steering settle questions around derivative trademarks and logos, Knative "extensions", and other questions about being part of our larger project. The KTC will be 'refreshed' in a year, at the same time that the 2021 steering elections happen. When the committee is refreshed, new vendors may receive a seat on the KTC, by virtue of their contributions in the previous year. Since we know that not all contributions are easy to quantify, any vendor may frame their contributions, and request consideration for membership on the KTC.
```

Comment:
```suggestion
The new "KTC" (Knative Trademark Committee) will initially include representatives from Google, Red Hat/IBM, and VMware. They will need to figure out how new components and features "plug in" to the Knative ecosystem in a non-technical sense. They will also help Steering settle questions around derivative trademarks and logos, Knative "extensions", and other questions about being part of our larger project. The KTC will be 'refreshed' in a year, at the same time that the 2021 steering elections happen. When the committee is refreshed, new vendors may receive a seat on the KTC, by virtue of their contributions in the previous year. Since we know that not all contributions are easy to quantify, any vendor may frame their contributions, and request consideration for membership on the KTC.
```